src/openrct2/ride/coaster/ReverseFreefallCoaster.cpp

```diff
@@ -385,6 +385,27 @@ static void paint_reverse_freefall_rc_vertical(
     }
 }
 
+static void reverse_freefall_rc_on_ride_photo(
+    paint_session* session, ride_id_t rideIndex, uint8_t trackSequence, uint8_t direction, int32_t height,
+    const TileElement* tileElement)
+{
+    static constexpr const uint32_t imageIds[4] = {
```

Comment:
```suggestion
    static constexpr const std::array<uint32_t, 4> imageIds = {
```

Reply:
Also, put a comment above explaining why we're using the station pieces here.